actions-server/src/handlers/joinGame.ts

```
@@ -81,9 +76,11 @@ const handler = async (req, res) => {
     exp: Math.floor(Date.now() / 1000) + 24 * 60 * 60,
   }
 
-  const token = jwt.sign(tokenContents, process.env.HASURA_GRAPHQL_JWT_SECRET)
+  const token = jwt.sign(
+    tokenContents,
+    process.env.HASURA_GRAPHQL_JWT_SECRET || "missing secret"
```

probably should raise an error here instead..